### PR TITLE
add resource blocks to k8s manifests

### PR DESCRIPTION
**Pull request by moosecanswim**
[sc-20865]

### CODE REVIEW

**Review by jack-bischoff — APPROVED** [review]

File: k8s/observables_deployment.yaml [hunk 1]
```diff
@@ -42,5 +42,12 @@ spec:
                 secretKeyRef:
                   name: elasticsearch-password
                   key: password
+          resources:
+            limits:
+              cpu: '1'
+              memory: 3Gi
+            requests:
+              cpu: 250m
+              memory: 1Gi
```
Comment (tilleryd) [inline k8s/observables_deployment.yaml:51]: I think you can match these with the dashboard resources in gitops-core.

Comment (moosecanswim) [reply k8s/observables_deployment.yaml]: Done

Comment (tilleryd) [reply k8s/observables_deployment.yaml]: they're different https://github.com/greymatter-io/gitops-core/blob/main/k8s/outputs/dashboard.cue#L53-L54

File: k8s/observables_deployment.yaml [hunk 2]
```diff
@@ -42,5 +42,12 @@ spec:
                 secretKeyRef:
                   name: elasticsearch-password
                   key: password
+          resources:
+            limits:
+              cpu: '2'
+              memory: 6Gi
+            requests:
+              cpu: 500m
+              memory: 2Gi
```
Comment (tilleryd) [inline k8s/observables_deployment.yaml:51]: why are these values different from the dashboard? https://github.com/greymatter-io/gitops-core/blob/main/k8s/outputs/dashboard.cue#L53-L54

File: k8s/observables_deployment.yaml (hunk 1 above)
Comment (moosecanswim) [reply k8s/observables_deployment.yaml:51]: Coolio

**Review by tilleryd — APPROVED** [review]

**Pull request merged**